#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_Makefile_rpath-fix.dpatch by Frederik Dannemare <frederik@dannemare.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove use of rpath in Makefile

@DPATCH@

--- motion-3.1.19/Makefile.orig	2005-03-02 20:53:24.115570880 +0100
+++ motion-3.1.19/Makefile	2005-03-02 20:53:48.423875456 +0100
@@ -31,7 +31,7 @@
 # install.                                                                     #
 ################################################################################
 CFLAGS       = -I/usr/include/ffmpeg -Wall -g -O2 -DHAVE_FFMPEG -DHAVE_XMLRPC -I/usr/include -DHAVE_MYSQL -I /usr/include/mysql -DHAVE_PGSQL -I/usr/include/postgresql -DHAVE_MEMMEM -DTYPE_32BIT="int"   -march=pentium4 -mfpmath=sse -msse2  -mmmx -DHAVE_BSWAP -Wall -DVERSION=\"$(VERSION)\" -D_REENTRANT -Dsysconfdir=\"$(sysconfdir)\"
-LDFLAGS      = -Wl,-rpath,/usr/lib
+LDFLAGS      = -Wl
 LIBS         =  -lm -lpthread -ljpeg -L/usr/lib -lavformat -lavcodec -lm -lz -L/usr/lib -lxmlrpc -lxmlrpc_xmlparse -lxmlrpc_xmltok  -L/usr/lib -lmysqlclient -lz -L/usr/lib -lpq 
 CONTROL_LIBS = -L/usr/lib -L/usr/lib -lxmlrpc_client -L/usr/lib -lwwwxml -lxmltok -lxmlparse -lwwwzip -lwwwinit -lwwwapp -lwwwhtml -lwwwtelnet -lwwwnews -lwwwhttp -lwwwmime -lwwwgopher -lwwwftp -lwwwfile -lwwwdir -lwwwcache -lwwwstream -lwwwmux -lwwwtrans -lwwwcore -lwwwutils -lmd5 -ldl -lz -lxmlrpc -lxmlrpc_xmlparse -lxmlrpc_xmltok  
 VIDEO_OBJ    = video.o
